pulse2percept/utils.py

```diff
@@ -42,7 +42,7 @@ def wrap(func):
         """
         e_s = "Call to deprecated function %s." % func.__name__
         if alt_func:
-            e_s += "Use %s instead." % alt_func
+            e_s += " Use %s instead." % alt_func
```

Comment:
It is set outside `wrap()` but inside `deprecated()` (starting l.57), in the case where `arg` is not callable:
```
alt_func = None
if callable(arg):
    # Direct decoration
    return wrap(arg)
else:
    # Alternative function given
    alt_func = arg
    return wrap
```